src/custom-apps/components/PermissionAlert/utils.ts

```diff
@@ -1,47 +1,32 @@
 // @ts-strict-ignore
```

Comment:
suggestion: If you rearrange a whole file would be good to remove it and fix types accordingly to rules